frontend/packages/kubevirt-plugin/integration-tests/tests/vm.wizard.scenario.ts

```diff
@@ -50,14 +56,23 @@ describe('Kubevirt create VM using wizard', () => {
   };
 
   beforeAll(async () => {
-    createResources([multusNAD, testDataVolume]);
+    createResources([testDataVolume]);
   });
 
   afterAll(async () => {
-    deleteResources([multusNAD, testDataVolume]);
+    deleteResources([testDataVolume]);
+  });
+
+  beforeEach(async () => {
+    createResources([multusNAD]);
   });
 
   afterEach(() => {
+    try {
+      deleteResources([multusNAD]);
```

Comment:
you can use singulars here: deleteResource etc.

Reply:
nop, I got an error when I tried

Reply:
that doesn't make sense? what was the error?